src/actions/hover.rs

```diff
@@ -642,9 +642,18 @@ fn racer_match_to_def(ctx: &InitActionContext, m: &racer::Match) -> Option<Def>
     let matchstr_len = matchstr.len() as u32;
     let docs = m.docs.trim().to_string();
     m.coords.map(|coords| {
+        // racer v2.1.0 seems to have a bug where it's populating the row
```

Comment:
I'd like to see this one solved before merging, especially the 2.1 seems fresh(?) and we can coordinate a fix with @kngwyu rather swiftly